src/error.rs

```diff
@@ -1143,16 +1143,25 @@ mod blame_error {
 
     /// Returns a title to be used by blame errors based on the `path` and `polarity`
     /// of the label.
-    pub fn title(l: &label::Label) -> &str {
+    pub fn title(l: &label::Label) -> String {
         if ty_path::has_no_arrow(&l.path) {
             // An empty path or a path that contains only fields necessarily corresponds to
             // a positive blame
             assert_eq!(l.polarity, Polarity::Positive);
-            "contract broken by a value"
+            match l.field_name {
+                Some(ident) => format!("contract broken by the value of {ident}"),
```

Comment:
For consistency, other message errors tend to quote identifiers coming from Nickel code, to distinguish them from a word of the error message.

```suggestion
                Some(ident) => format!("contract broken by the value of `{ident}`"),
```